se.trixon.Mapollage.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>se.trixon.Mapollage</id>
+    <name>Mapollage</name>
+    <project_license>Apache-2.0</project_license>
+    <metadata_license>CC0-1.0</metadata_license>
+    <content_rating type="oars-1.1" />
+    <developer_name>Patrik Karlström</developer_name>
+    <summary>A photo kml generator for Google Earth</summary>

```

Comment:
```suggestion
    <summary>A photo kml generator for Google Earth</summary>

```

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#dont-start-with-an-article

Reply:
It's changed.